src/thunderbots/software/backend/output/radio/mrf/mrf_primitive_visitor.h

```diff
@@ -1,9 +1,10 @@
 #pragma once
 
+#include <array>
```

Comment:
why do we need this?

Reply:
we use `std::array` in this file

Reply:
Confusing why we didn't need this before but ok